File: LP/C/ex3.c
#include <stdio.h>
#include <string.h>

void main(){
    int num;

    printf("Digite um número: ");
    scanf("%d", &num);

    printf("Os primeiros %d números ímpares são: ", num);
    for (int i = 1; num > 0; i++){
        if (i % 2 == 1){
            printf("%d ", i);
            num--;
        }
    }
    printf("\n");
}

File: LP/GO/ex3.go
package main

import(
	"fmt"
)

func main(){
	var(
		num int
		cont = 0
	)

	fmt.Print("Digite o quantos números ímpares quer ver: ")
	fmt.Scan(&num)
	fmt.Printf("Os primeiros %d números impares são: ")

	for i := 1; cont < num; i++{
		if i % 2 == 1{
			fmt.Printf("%d, ", i)
			cont += 1
		}
	} 
}